docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.7'
```

Comment:
You can just add script something like it
```"docker run --rm -it -p 5432:5432 --name gamesbe -e POSTGRES_PASSWORD=root -e POSTGRES_USER=root -e POSTGRES_DB=games.db postgres:13"```

Reply:
You're right but as I'm also using 'volumes', the run command will look too long. That's why I created a docker-compose file.